src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partner;
+
+  if (isMarried) {
```

Comment:
This should also be extracted into a separate helpers file as a helper function, like `generatePatnerDescription(person)`.

Could you also think how to simplify this function?